ethereum.py

```diff
@@ -17,15 +18,15 @@ def get_ethereum_balance(apiprovider, wallet, rpc_call_status_counter):
             rpc_call_status_counter.labels(
                 url=apiprovider, status=MetricsUrlStatus.SUCCESS.value
             ).inc()
-            return erc20_balance
+            return {"balance": erc20_data["balance"], "symbol": erc20_data["symbol"]}
         else:
             web3 = Web3(Web3.HTTPProvider(apiprovider))
             balance = web3.eth.get_balance(addr)
             balance_ether = web3.from_wei(balance, "ether")
             rpc_call_status_counter.labels(
                 url=apiprovider, status=MetricsUrlStatus.SUCCESS.value
             ).inc()
-            return balance_ether
+            return {"balance": balance_ether, "symbol": "ETH"}
```

Comment:
it's not always ETH here. for instance:
on moonbeam : that will be GLMR
on harmony ONE : that will be ONE
a default to ETH why not, but for some chain it won't be ETH